src/nodes/footstep_planner_server

```diff
@@ -46,7 +47,18 @@ class ActionServ(object):
 
         # self.footStepPlan = FootstepPlannerSl1m()
         # self.cost = {"final_com": [1.0, [0.0,0.0,0.0]]}
-        self.sl1m_node = Sl1mNode()
+        self.sl1m_nodes = [Sl1mNode(), Sl1mNode()] # for starting with left feet or right feet
```

Comment:
@pFernbach here you should just swap the gait parameter...
That should be enough to start with the correct foot.